minimal/compose/nginx/production/nginx.conf

```diff
@@ -36,6 +36,10 @@ http {
         listen 80;
         charset     utf-8;
 
+        location /.well-known  {
+            alias /code/staticfiles/.well-known;
```

Comment:
Don't think it's necessary. Anything in .well-known is very short lived (Certbot creates a temporary file, checks to see if it can reach that file and then deletes everything again so no need for cache.